web/css/source/variables/custom/_colors.less

```diff
@@ -2,3 +2,7 @@
 @color-orange-mixed-with-white: #ffb89e;
 @color-new-awesome-orange: #e8a535;
 @color-new-awesome-green: #0B5563;
+@color-vaimo-yellow: #fabf2b;
+@color-vaimo-yellow-active: #f0af19;
+@color-text-black: #333;
+@color-main-white: white;
```

Comment:
if it has to be used for header and main navigation only, call it with the explanation like
```less
@color-header-cta: white;
```

Reply:
oh, I got that. u used it only for the disabled state of input and textarea.
so, create another one for header CTAs

Reply:
No I use it like a general color all over the website. I can change the name to like @color-general-white or something